src/ui/public/timepicker/timepicker.less

```diff
@@ -0,0 +1,19 @@
+.kbn-timepicker {
+  [kbn-time-input] {
+    text-align: center;
+  }
+  .kbn-timepicker-modes {
+    text-transform: capitalize;
+  }
+  .kbn-timepicker-section {
+    float: left;
+    padding: 0px 15px;
+  }
+  .kbn-refresh-section {
+    float: left;
+    padding: 0px 15px;
+  }
+  .kbn-timepicket-alert {
+    width: 100px;
+  }
+}
```

Comment:
I agree that we should limit the scope of these classes to only the kbn-timepicker. We don't want to use them outside of this component. I'm convinced; we can keep them the way they are for now. :)